### DISCUSSION TITLE
Installer (Windows) - "Uninstall Rustdesk" not translated

### DISCUSSION THREAD

**Discussion by bovirus** [rustdesk/rustdesk#4574] [Feature Request]
### Bug Description

The program grpoup contain a "Uninstall Rustdesk" icon description.

The word "Uninstall" is untranslated.

### How to Reproduce

Uninstall the beta 1.2.0 in Windows

### Expected Behavior

Add the word "Uninstall Rustdesk" in language file and use it.

### Operating system(s) on local side and remote side

Windows 10 64bit 22H2

### RustDesk Version(s) on local side and remote side

1.2.0 beta 06.06.2023

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/1262554/10e976e3-447e-4645-93cf-284d92a1cba7)

**Comment by rustdesk** [comment]
Frankly, we have no plan to translate this yet.

**Comment by bovirus** [comment]
@rustdesk 

The string is in file **src/platform/windows.rs**

**Uninstall** {app_name}.lnk
